docs/gettingstarted/installation-macos.rst

```diff
@@ -18,7 +18,18 @@ PostgreSQL
 
 Saleor needs PostgreSQL version 9.4 or above to work. Get the macOS installer from the `PostgreSQL download page <https://www.postgresql.org/download/macosx/>`_.
 
-Make sure you keep track of the password you set for the administration account during installation.
+Then, create a new database and user. Make sure you keep track of the password you set for the administration account during installation. 
```

Comment:
Maybe you could change to "Then, you **will** need to create a new database and user.", letting know the user further instructions and code blocks are coming.